traits/ctraits.c

```diff
@@ -5021,6 +5021,65 @@ PyMODINIT_FUNC PyInit_ctraits(void) {
                          (PyObject *) &trait_type ) < 0 )
        return NULL;
 
+    /* Expose module-level constants used by cTrait and CHasTraits */
+    /* CHasTraits flags */
+    PyModule_AddIntConstant(module, "_HASTRAITS_INITED", HASTRAITS_INITED);
+    PyModule_AddIntConstant(
+        module, "_HASTRAITS_NO_NOTIFY", HASTRAITS_NO_NOTIFY);
+    PyModule_AddIntConstant(
+        module, "_HASTRAITS_VETO_NOTIFY", HASTRAITS_VETO_NOTIFY);
+
+    /* cTrait flags */
+    PyModule_AddIntConstant(module, "_TRAIT_PROPERTY", TRAIT_PROPERTY);
```

Comment:
I think adding these only makes sense if we also intend to add a corresponding `IntEnum` subclass for the cTrait flags. It may make sense to do that in this PR, or perhaps separate out these changes and make a single PR for all the flag-related things.